lesson_l16/task_01/developer.py

```diff
@@ -0,0 +1,5 @@
+from lesson_l16.task_01.employee import Employee
+
+
+class Developer(Employee):
+    programming_language: str
```

Comment:
such attributes is better to be private, and use getter methods to provide access for that.

Reply:
Updated in commit 32fb69f57063920828ca5fa16dcb7698619bada9